dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -102,14 +166,48 @@ postinstall(){
     rosa edit cluster -c $(_get_cluster_id ${CLUSTER_NAME}) --expiration=${EXPIRATION_TIME}
 }
 
+index_metadata(){
+    DURATION=$(($2 - $1))
+    ACTION=$3
+    if [ ${ACTION} == "ready" ] || [ ${ACTION} == "Installation Finished" ] ; then
+        _download_kubeconfig $(_get_cluster_id ${CLUSTER_NAME}) ./kubeconfig
+        export KUBECONFIG=./kubeconfig
+    fi
+    local METADATA=$(cat << EOF
+{
+"uuid" : "${UUID}",
+"platform": "ROSA",
+"network_type": "${NETWORK_TYPE}",
+"aws_authentication_method": "${AWS_AUTHENTICATION_METHOD}",
+"cluster_version": "$(oc version -o json 2>/dev/null | jq -r .openshiftVersion)",
+"master_count": "$(oc get node -l node-role.kubernetes.io/master= --no-headers 2>/dev/null | wc -l)",
+"worker_count": "$(oc get node -l node-role.kubernetes.io/worker=,node-role.kubernetes.io/workload!=,node-role.kubernetes.io/infra!= --no-headers --ignore-not-found 2>/dev/null | wc -l)",
+"infra_count": "$(oc get node -l node-role.kubernetes.io/infra= --no-headers --ignore-not-found 2>/dev/null | wc -l)",
+"workload_count": "$(oc get node -l node-role.kubernetes.io/workload= --no-headers --ignore-not-found 2>/dev/null | wc -l)",
+"total_node_count": "$(oc get nodes 2>/dev/null | wc -l)",
+"ocp_cluster_name": "$(oc get infrastructure.config.openshift.io cluster -o json 2>/dev/null | jq -r .status.infrastructureName)",
+"rosa_cluster_name": "${CLUSTER_NAME}",
+"action": "${ACTION}",
```

Comment:
perhaps name this "state" to be consistent with what the rosa cli reports

Reply:
Perf team is already using action since time ago, if we want to compare results, we need to use the same parameter

Reply:
moved to managedservices-install-timings index